site/pt/blog/new-in-devtools-96/index.md

```diff
@@ -29,166 +29,213 @@ draft: true
 {% include 'partials/devtools/pt/banner.md' %}
 
 <!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
-
+## Prévia de funcionalidade: novo painel de visão geral CSS  {: #css-overview }
 <!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
 [Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
 
+Use o novo painel **Visão geral CSS** para identificar possíveis melhorias de CSS em sua página.
+[Abra o painel **Visão geral CSS**](/docs/devtools/css-overview#open) e clique em **Capture visão geral** para gerar um relatório do CSS da sua página.
+
 <!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Você pode detalhar ainda mais as informações. Por exemplo, clique em uma cor na seção **Cores** para visualizar a lista de elementos que aplicam a mesma cor. Clique em um elemento para abri-lo no painel **Elementos**.
 
 <!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+O painel **Visão geral CSS** é um recurso em desenvolvimento. Nossa equipe ainda está trabalhando ativamente nele e estamos atentos aos seus [comentários](https://goo.gle/css-overview-feedback) para mais melhorias.
 
 <!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+Leia [este artigo](/docs/devtools/css-overview) para saber mais no painel **Visão geral CSS**.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="Painel Visão Geral CSS", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
 Issue relacionada: [1254557](https://crbug.com/1254557)
 
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
```

Comment:
Hi @alvarocamillont , I added a new section here (a fix we recently merged back to M96). Sorry for late inform!